tests/core.module.spec.js

```diff
@@ -7,5 +7,7 @@
   function mock($httpBackend) {
     $httpBackend.when('GET', /available_languages.json/)
       .respond({});
+    $httpBackend.when('GET', /notifications/)
```

Comment:
Aah, can you make this `/api\/notifications/`? We do have a few files with `notifications` in their path...